src/robusta/runner/main.py

```diff
@@ -24,21 +27,20 @@ def main():
     event_handler = PlaybooksEventHandlerImpl(registry)
     loader = ConfigLoader(registry, event_handler)
 
-    if ENABLE_TELEMETRY:    
-        telemetry_service = TelemetryService(
-            telemetry_level= TelemetryLevel.ERROR if SEND_ADDITIONAL_TELEMETRY else TelemetryLevel.USAGE,
+    if ENABLE_TELEMETRY:
+        TelemetryService(
```

Comment:
Will this cause the GC to clear this object out of scope?
Verify telemetry still is working out of scope

Reply:
I thought about it and as I remember looking inside - it is handling some threads that will have a reference on it while working - so GC will not remove it